### DISCUSSION TITLE
Any real quality face swapping ?

### DISCUSSION THREAD

**Discussion by Coolzeroalpha** [lllyasviel/stable-diffusion-webui-forge#590] [Q&A]
Hello,
Is there a good method for doing face swapping? 

So far, all the methods I've seen and tried haven't been great. ReActor, ControlNet, IpAdapter, I also tried Easy Photo to do a Lora and the results aren't satisfactory. 

Basically, yes, you can see that it resembles me quite a bit, there's clearly a strong resemblance, but in no way can you believe it's me

**Comment by GrooveIsNow** [comment]
Why not try instant-id? It's built in the controlnet, just have to install the models. Works very well for me, as most face swapping methods can't do my likeness, this one works great! https://github.com/Mikubill/sd-webui-controlnet/discussions/2589

**Reply by Coolzeroalpha** [reply]
Thank you :)
I wasn't aware of Instant-ID. Do you have any recommendations on how to use it for optimal results ? Maybe a tutorial or YT video.
I'm aiming to create realistic pictures where even those who know me won't doubt it's really me. For example, pictures of me at a fancy restaurant or at the top of Mt. Fuji
